calendar/endpoints.bal

```diff
@@ -166,12 +166,14 @@ public client class Client {
     # 
     # + calendarId - Calendar id
     # + userAccount - The email address of the user for requesting delegated access in service account
+    # + optional - Record that contains optional parameters
     # + return - Event stream on success, else an error
     @display {label: "Get Events"}
     remote isolated function getEvents(@display {label: "Calendar Id"} string calendarId,
+                                        @display {label: "Events to Access"} ListEventsToAccess? optional = (),
```

Comment:
`Filtering Criteria`

Reply:
Fixed in https://github.com/ballerina-platform/module-ballerinax-googleapis.calendar/commit/67761b55d161fbf36993eadfc4b8ee43652cc3c9